recipes/Genesis-PyAPI/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "Genesis-PyAPI" %}
+{% set version = "1.2.1" %}
+{% set sha256 = "2989f582eb619cd18c0fda10c69d38404781ddc4161a77f18c29d02fbdee577c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py2k]
```

Comment:
can this be a noarch:python package ?

According to @CJ-Wright, you can remove the skip line and add a python >=3.5 run dependency.